grafana-plugin/src/containers/IntegrationContainers/ExpandedIntegrationRouteDisplay/ExpandedIntegrationRouteDisplay.tsx

```diff
@@ -89,6 +105,10 @@ const ExpandedIntegrationRouteDisplay: React.FC<ExpandedIntegrationRouteDisplayP
     const channelFilterIds = alertReceiveChannelStore.channelFilterIds[alertReceiveChannelId];
     const isDefault = IntegrationHelper.getRouteConditionWording(channelFilterIds, routeIndex) === 'Default';
 
+    if (isLoading) {
+      return <LoadingPlaceholder text="Loading..." />;
```

Comment:
in general it's better to display some stub instead of displaying just LoadingPlaceholder